package.json

```diff
@@ -44,6 +44,7 @@
     "preact-render-to-string": "^5.1.11",
     "prettier": "^2.4.1",
     "rollup": "^2.38.0",
+    "rollup-plugin-node-polyfills": "^0.2.1",
```

Comment:
Why did this get added? I don’t see it being used in the rollup config.

Generally, I think these types of plugins are a footgun. If there are any node packages that need polyfilling, as they are often unnecessarily large for the sake of fidelity that is rarely needed. If we need something, let’s see if we can find a different workaround.

Reply:
Oof sorry, the PR was in a more ready state a little bit before you came. This morning I experimented a little bit with different zip libraries and some of them broke rollup so I tried to fix it. I'll remove this in a bit.